maintainers-guide.md

```diff
@@ -195,34 +195,40 @@ issues.
 *   [`team-Android`](https://github.com/bazelbuild/bazel/labels/team-Android): Issues for Android team
     *   Contact: [ahumesky](https://github.com/ahumesky)
 *   [`team-Apple`](https://github.com/bazelbuild/bazel/labels/team-Apple): Issues for Apple team (also MacOS, iOS etc)
-    * Contact: [aragos](https://github.com/aragos)
+    * Contact: ?
```

Comment:
I don't think he would agree with the description here. jmmv@ should get issues related to running Bazel on macos.  Issues about building for iOS and macos belong to the apple rules team, or sometimes the CPP team.
@jmmv do you want to suggest a description rewrite?

Reply:
You can put me here, yes.

And, yes, agreed. This is stale and I'd even say the name is misleading at this point. I'd rather have `team-macOS` or `team-Mac` to clearly say that this is not a bucket for all Apple-related issues. There is no "Apple team" any longer.

The question, though, is where do we fit iOS rule issues. Some of them belong to in the `rules_apple` repository (though we cannot move issues). Some of them belong in the C++ rules as they share some code. But we still have some rule code in Bazel around this...

Reply:
> Some of them belong to in the rules_apple repository (though we cannot move issues).

Why not? As long as the issue mover has permissions to manage issues in rules_apple.

We can take this opportunity to mass-update team-Apple issues to team-macOS.

Reply:
Wait, what? I had never seen the `*Transfer issue*` option! That's nice.

Reply:
I'll update once the labels are updated (https://github.com/bazelbuild/bazel/labels?page=3&sort=name-asc). For now, I prefer to keep the list in sync with the actual labels and their description.

Reply:
Seems easier for whoever is submitting this PR to also rename the labels at the same time (assuming the right permissions...). Otherwise there will always be skew between the documentation and the labels.

But I agree that renaming the labels should happen separately.

Reply:
I'm not qualified for making changes to the labels. I just want to make sure this page is up-to-date.

I think anyone in the Bazel team can edit the labels (but should talk with the people from the DevEx rotation to ensure good triaging).

Reply:
I'm happy to help coordinate with @jmmv to mass re-label the affected issues and PRs.